CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -30,12 +30,21 @@ public class ChromiumWebBrowser : Control, IWebBrowserInternal, IWinFormsWebBrow
         /// The browser
         /// </summary>
         private IBrowser browser;
+        /// <summary>
+        /// A flag that indicates whether or not the designer is active
+        /// </summary>
+        private bool designMode;
```

Comment:
We do, sadly DesignMode becomes false by the time we get to the destructor/dispose